README.md

```diff
@@ -18,6 +18,47 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
 
 # Tips list
 
+## #0xx - "use strict" and get lazy
+
+> 2016-xx-xx by [@nainslie](https://twitter.com/nat5an)
+
+By default, JavaScript allows the programmer to be pretty careless, for example, by not requiring us to declare our variables with "var" when we first introduce them.  While this may seem like a convenience to the unseasoned developer, it's also the source of many errors when a variable name is misspelled or accidentally referred to out of its scope.
+
+Programmers like to make the computer do the boring stuff for us, and automatically check our work for mistakes. That's what the JavaScript "use strict" directive allows us to do, by turning our mistakes into JavaScript errors. 
+
+We add this directive either by adding it at the top of a js file: 
+
+```javascript
+// Whole-script strict mode syntax
+"use strict";
+var v = "Hi!  I'm a strict mode script!";
+```
+
+or inside a function:
+
+```javascript
+function strict()
```

Comment:
It will be good to change the name of the function to something other than 'strict'. There are high chances for the newbies to get confused with 'use strict'